source/docs/v3/tenant-attribute-encryption.blade.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Tenant attribute encryption
+extends: _layouts.documentation
+section: content
+---
+
+# Tenant attribute encryption {#encrypt}
+
+To encrypt attributes on the Tenant model, store them in [custom columns](https://tenancyforlaravel.com/docs/v3/tenants/#custom-columns) and cast the attributes to `'encrypted'`, or your custom encryption cast.
```

Comment:
See how we create links in all other pages of the docs and follow the same pattern. We don't want to hardcode the URL or "v3" here